src/shutdown/umount.c

```diff
@@ -172,12 +168,60 @@ int mount_points_list_get(const char *mountinfo, MountPoint **head) {
 
                 TAKE_PTR(remount_options);
 
+                unmountable = !nonunmountable_path(m->path);
+
+                // Order matters. We will remount from deep to root.
```

Comment:
as per coding style: only use `/* */` style comments for explanatory stuff, and `//` for FIXME style stuff